doc/source/user_guide/machine_learning_pysteps.rst

```diff
@@ -0,0 +1,103 @@
+.. _machine_learning_pysteps:
+
+Machine learning vs pysteps
```

Comment:
not quite convinced by the title "Machine learning vs pysteps" for two main reasons:

* it sounds a bit like a competition between a python package and the whole machine learning ...., while here is more a matter of benchmarking machine learning with traditional persistence based approaches available in pysteps.
* in principle, pysteps represents a general framework/community for precipitation nowcasting, and as such we could imagine to implement ML-based models, too. 

For these reasons, I would propose to change the title into something a bit different, as for example "Benchmarking machine learning models with pysteps". 

What do you think?